directorystructure.md

```diff
@@ -0,0 +1,28 @@
+# Directory Structure
```

Comment:
Let's please put this at the top of the existing code description repository instead of a new file.